app/controllers/api/v0/reading_collections_controller.rb

```diff
@@ -0,0 +1,18 @@
+module Api
+  module V0
+    class ReadingCollectionsController < ApiController
+      def create
+        reading_collection = current_user.reading_collections.create(collection_params)
+        articles = reading_collection.get_articles
+        reading_collection.articles << articles
+        @reading_collection = reading_collection.to_json
```

Comment:
I see this inherits from the ApiController what does that allow it access to, or what's the value in that inheritance? 

Just trying to get a sense of what's going on.

Reply:
It's "rails magic"! Basically, it gives it some methods that I don't even fully understand :)

Reply:
Good question! For APIs, we usually inherit everything this way as I believe it's for versioning purposes. If there is a future version change, we'll basically make a new folder (/api/v1) and make updates. Then change routes to pull from updated material.